.golangci.next.reference.yml

```diff
@@ -637,9 +622,6 @@ linters-settings:
         # The option is passed to the ruleguard 'debug-group' argument.
         # Default: ""
         debug: 'emptyDecl'
-        # Deprecated, use 'failOn' param.
-        # If set to true, identical to failOn='all', otherwise failOn=''
-        failOnError: false
```

Comment:
was noticed that `jsonschema/golangci.next.jsonschema.json` doesn't describe the gocritic settings in details, just the checkers list 

but checkers list is not right solution too, because not all checkers have settings 🤔

Reply:
I updated the JSON schema.